lib/tasks/data/truncate_data.rake

```diff
@@ -9,10 +9,11 @@ require 'tasks/data/truncate_data'
 namespace :ofn do
   namespace :data do
     desc 'Truncate data'
-    task truncate: :environment do
+    task :truncate, [:months_to_keep] => :environment do |_task, args|
       guard_and_warn
 
-      TruncateData.new.call
+      months_to_keep = args.months_to_keep.to_i
+      TruncateData.new(months_to_keep: months_to_keep).call
```

Comment:
can be just       TruncateData.new(months_to_keep).call

Reply:
Yes, things evolved since I introduced the keyboard argument and now I see this is just redundant :+1: